pdfjadetex should default to pdf mode, not dvi mode.
Upgrade to the .ini file as shipped in current TeX Live
---
 jadetex.ini    |    7 ++++++-
 pdfjadetex.ini |   11 ++++++++++-
 2 files changed, 16 insertions(+), 2 deletions(-)

Index: jadetex-3.13/pdfjadetex.ini
===================================================================
--- jadetex-3.13.orig/pdfjadetex.ini	2001-07-03 04:52:12.000000000 +0900
+++ jadetex-3.13/pdfjadetex.ini	2010-02-13 12:30:21.000000000 +0900
@@ -1,3 +1,12 @@
+% public domain.
+% 
+% do what we do for pdflatex, but don't dump now.
+\let\primitivedump=\dump
+\let\dump=\relax
+\input pdflatex.ini
+\let\dump=\primitivedump
+%
+% then read jadetex.  pdf output must be set first, since it needs hyperref.
 \input jadetex.ltx
+%
 \dump
-\endinput
Index: jadetex-3.13/jadetex.ini
===================================================================
--- jadetex-3.13.orig/jadetex.ini	2010-02-13 12:31:34.000000000 +0900
+++ jadetex-3.13/jadetex.ini	2010-02-13 12:31:43.000000000 +0900
@@ -1,3 +1,8 @@
+% public domain.
+\let\primitivedump=\dump
+\let\dump=\relax
+\input latex.ini
+\let\dump=\primitivedump
+
 \input jadetex.ltx
 \dump
-\endinput
\ No newline at end of file
